src/views/tls/key-exchange/index.js

```diff
@@ -0,0 +1,74 @@
+import {api} from '../../../client';
```

Comment:
nit: Alphabetize

I know, it's not clear, so I will add a special configuration in the future to have a standard enforced by our linter, but I have to talk with Livvie or Carlos first. No action required.